app/views/layouts/provider/login.html.slim

```diff
@@ -17,4 +17,5 @@ html lang="en"
     # are split Provider/Buyer side
 
   body class="login-layout"
+    = javascript_pack_tag 'LoginPage'
```

Comment:
This is ok since there's just one layout - page affected, either way you should need to yield something like `:javascript_login` and se `content_for` helper in the views